### PR TITLE
Ne xdr prevent only mode

### PR DESCRIPTION
**Pull request by nericksen**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by JasBeilin** [comment]
Hi @nericksen.
Could you please provide some context to this change? Why is it needed?

**Comment by nericksen** [comment]
When dealing with a Prevent only XDR tenant the integration will not work without this fix

**Review by JasBeilin — CHANGES_REQUESTED** [review]
Thank you for contributing!
Please check my comments so we can move forward :)

File: Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml [hunk 1]
```diff
@@ -91,6 +91,11 @@ configuration:
   name: proxy
   required: false
   type: 8
+- display: Prevent Only Mode
+  name: prevent_only
+  required: false
+  type: 8
+  additionalinfo: 'Select when the XDR tenant is prevent only'
```
[the remark this answers is not in the document]
Comment (nericksen) [reply Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml]: Updated thanks! Let me know if anything else is needed

**Review by JasBeilin — APPROVED** [review]

**Pull request merged**